### ISSUE TITLE
[DataCap Application] EMPIAR Public dataset

### ISSUE THREAD

**Issue by gimims** [TOPPOOL-LEE/Allocator-Pathway-TOP-POOL#22]
### Version

1

### DataCap Applicant

FileTech

### Project ID

FileTech-03

### Data Owner Name

EMPIAR Public dataset

### Data Owner Country/Region

United Kingdom

### Data Owner Industry

Life Science / Healthcare

### Website

https://www.ebi.ac.uk/empiar

### Social Media Handle

https://www.ebi.ac.uk/empiar

### Social Media Type

Slack

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

1PiB

### Number of replicas to store

5

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f17qa7psscfah4rnyxcfhxj24v5wjp36wjsvilgiq

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
FileTech focuses on providing excellent data storage solutions. We have a passionate and knowledgeable team with extensive experience and expertise in the field of data storage. Whether it's data storage, data management, data recovery, or data center design and construction, we possess abundant technical capabilities and solutions.

At FileTech, we understand the importance of data in modern businesses. We not only offer high-performance data storage devices and solutions, but also provide comprehensive data management tools to help clients efficiently organize, classify, and protect their data assets.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
EMPIAR, the Electron Microscopy Public Image Archive, is a public resource for raw images underpinning 3D cryo-EM maps and tomograms (themselves archived in EMDB). EMPIAR also accommodates 3D datasets obtained with volume EM techniques and soft and hard X-ray tomography. More ...
As of 2023-03-27, EMPIAR contains 1254 entries, taking up 2.73 PB of storage.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

France

### Please share a sample of the data

```text
Yes
https://www.ebi.ac.uk/empiar
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America, Europe

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), IPFS, Shipping hard drives

### How did you find your storage providers

Slack, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f01907578  Shanghai
f02870401   Jakarta
f03144037   Seoul, KR
f02894286   Seoul, KR
f01111110   Hanoi
f01909705   Hanoi
f02832475   Thailand
```

### How do you plan to make deals to your storage providers

Boost client, Lotus client, Bidbot

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by TOPPOOL-LEE** [comment]
Thanks for your application:

Please tell me if you have relevant experience and how you will prepare the data.
Why did you choose this data set.
How much data have you downloaded? Do you need our help?
Will you save unseal and support spark?

**Comment by TOPPOOL-LEE** [comment]
Please list the SP, company name and geographical location in detail, thank you!

**Comment by gimims** [comment]
Yes, we have many years of experience in data storage, cutting, distribution and preservation, and we are familiar with DC rules and requirements. We have chosen a data set with increasing data, and we want to store the new data on the Filecoin network. We have downloaded 100T of data and are continuing to download it. We require the cooperating sp to save the file and satisfy spark.

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger

**Comment by TOPPOOL-LEE** [comment]
you need to explain

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger

**Comment by TOPPOOL-LEE** [comment]
It seems that the data is not updated.

Why don't you reply?

**Comment by gimims** [comment]
Dear allocator, I'm sorry,  the company team was traveling, so I didn't see the news in time.Our technical team is very familiar with the rules of FIL+ and  Spark, so we will support Spark and be searchable. So, could you  help us sign a round?

**Comment by gimims** [comment]
checker:manualTrigger

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger

**Comment by gimims** [comment]
Dear allocator, as we stated on the phone: there have been some updates to our sp, so I have modified the application form and hope to get your approval, thank you.

**Comment by gimims** [comment]
We know that you are very concerned about Spark , so we will pay close attention to SP and we require all SPs we cooperate with to support spark.

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger

**Comment by TOPPOOL-LEE** [comment]
<img width="1146" alt="WX20240819-134051@2x" src="https://github.com/user-attachments/assets/7fb36fff-f2e1-460f-a665-eebef12147c4">

**Comment by TOPPOOL-LEE** [comment]
Please tell me the reason why the retrieval rate of these three sp is 0. @gimims

**Comment by gimims** [comment]
checker:manualTrigger

**Comment by gimims** [comment]
<img width="654" alt="WechatIMG225" src="https://github.com/user-attachments/assets/b9c06cb6-6075-4d7a-9206-4629eacae77a">

**Comment by gimims** [comment]
There are the recently launched SPs. We checked the background and found that the SP did not receive a retrieval request, but we support retrieval.We are working hard to follow Miroslav Bajtoš (SpaceMeridian) communicated.
We hope you will approve the next round for us.

**Comment by gimims** [comment]
thanks

**Comment by gimims** [comment]
checker:manualTrigger

**Comment by gimims** [comment]
We have finished using it. Can you help us?

**Comment by gimims** [comment]
We have checked all the reasons. The boost service is normal, the index is open, and the retrieved files are stored. What is the problem about the spark？

**Comment by gimims** [comment]
hope to get help from the officials

**Comment by gimims** [comment]
checker:manualTrigger

**Comment by gimims** [comment]
checker:manualTrigger

**Comment by TOPPOOL-LEE** [comment]
@gimims  Sorry for not replying to you on GitHub.
I have already told you the reason why I did not continue to sign for you through the tg conference call last month, and sent a technician to tell you how to improve the spark data.
I hope you can reply here, thank you. @gimims

**Comment by gimims** [comment]
@TOPPOOL-LEE 
We are glad that Toppool signed DC for us and sent technicians to help guide the work when SP encountered spark problems. With the help we found the spark guys on slack.
We communicate on GitHub, Slack, TG.
In August, you called us on TG and told us that the sp retrieval success rate on Spark was very low, it was not possible to sign .
Thank you, we will continue to work hard to improve the retrieval success rate about spark.

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger

**Comment by TOPPOOL-LEE** [comment]
Hello, we suddenly found that all our cilents have too much duplicate data stored. Is this a coincidence or a bug? We can understand that one customer has this problem, but it seems a bit strange that four cilents have this problem at the same time, especially four cilents who have never had this problem before. Can you help us check it out? @willscott@bajtos · he/him

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger

**Comment by gimims** [comment]
检查器：手动触发

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger

**Comment by TOPPOOL-LEE** [comment]
https://github.com/fidlabs/allocator-tooling/issues/73

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger

**Comment by TOPPOOL-LEE** [comment]
checker:manualTrigger
